### PR TITLE
allow context to be set

### PR DESCRIPTION
**Pull request by phiros**
Allows the status context to be configured for the status plugin.
More details about status contexts may be found here: https://developer.github.com/changes/2014-03-27-combined-status-api/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   },
   "homepage": "https://github.com/Strider-CD/strider-github-status",
   "dependencies": {
-    "github": "~0.1.12",
+    "github": "~0.2.4",
     "debug": "~0.7.4"
```
Comment (knownasilya) [inline package.json]: Could you change both of these to carrots `^`? Thanks!

File: webapp.js [hunk 2]
```diff
@@ -13,12 +13,17 @@ function jobDescription(job) {
 }
 
 module.exports = {
+  config: {
+    pending.msg: String,
```
Comment (knownasilya) [inline webapp.js]: Is that `.` valid without wrapping it in a string?

**Comment by phiros** [comment]
Ah that was silly. Should have run this thing first before creating the PR.
Will change this.

File: lib/handler.js [hunk 3]
```diff
@@ -17,8 +17,14 @@ if (GITHUB_API_ENDPOINT) {
...
 
-module.exports = function (token, url, data, status, description) {
-  debug('Setting status', token, url, data, status, description)
+module.exports = function (token, url, data, status, description, context) {
+  debug('Setting status', token, url, data, status, description, context)
+  if(description === undefined) {
+    description = 'Strider test in progress';
+  }
+  if(context === undefined) {
+    context = 'strider-cd';
```
Comment (knownasilya) [inline lib/handler.js]: Per that link, looks like the recommended context format is 'ci/strider'.

Comment (phiros) [inline lib/handler.js]: Hmm, well this blog post is a little bit old.
Travis for example sets the following context:
 continuous-integration/travis-ci/pr

Comment (knownasilya) [inline lib/handler.js]: The more specific the better I think, just in case we have other contexts later.

**Comment by knownasilya** [comment]
Is this all set for merging? If so, could you squash your commits? Thanks!

**Comment by leoetlino** [comment]
I've tried this PR and it seems that it doesn't work entirely, since I cannot configure it (I just get "This plugin does not require any configuration" in the web UI).

**Comment by knownasilya** [comment]
Closing in favor of #11

**Pull request closed without merging**